projects/tuvali.md

```diff
@@ -0,0 +1,93 @@
+# Project Name
+Tuvali - Means a tunnel in [Tamil](https://en.wikipedia.org/wiki/Tamil_language)
+
+# Project Description
+This library contains the implementation for Android & IOS for the Openid4BLE specification. It's written in Kotlin & Swift.
+
+# Alignment with the OpenWallet Foundation Mission
+This library implements the BLE-based local transfer of Verifiable Presentation (VP). An offline-based mode provides us with the ability to present a VP in many crises and crowded situations, from stadium entries to social benefits in remote areas. This is an important building block for future digital identity wallets to be inclusive. Since it is written in Kotlin & Swift, it can be used widely.
+
+# Current Code of Conduct
+[Code of Conduct](https://docs.mosip.io/1.2.0/community/code-of-conduct)
+
+We will also follow the LF Code of Conduct
+
+# TAC Sponsor
+No one Now
+
+# Project License
+Currently, the project is hosted with MPL 2.0 license. We will move to Apache license 2.0 upon approval of the project.
+
+# Source Control
+[Repository](https://github.com/mosip/tuvali)
+
+# Issue Tracker
+[Jira](https://mosip.atlassian.net/issues/?jql=labels%20%3D%20%22BLE%22)
+
+# External Dependencies
```

Comment:
Are all the dependencies compatible with Apache license?

Reply:
Yes, We have listed the dependencies and their license.